nbs/tutorials/extensions.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e457d9df-9dc0-40ba-a4b9-107a3de72556",
+   "metadata": {},
+   "source": [
+    "# Writing nbdev plugins\n",
+    "\n",
+    "> How to customize nbdev processors to do what you want\n",
+    "\n",
+    "- order: 9"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7dd89fcf-a6a8-4ae7-bb79-4b81f0586211",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
+    "from __future__ import annotations\n",
+    "from fastcore.test import *\n",
+    "from nbdev.showdoc import *\n",
+    "from nbdev.qmd import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "521da717-490e-473c-8d4c-c0ae0db3061d",
+   "metadata": {},
+   "source": [
+    "## What will this cover?\n",
+    "\n",
+    "With `nbdev` it is *exceedingly* easy to customize it to your own doing. Does your particular library or setup require you to inject some custom quarto bits in certain cells? What about if you want to do something more trivial as finding shortcuts to replace the `---` annotation used in quarto. \n",
+    "\n",
+    "Writing custom plugins is exactly how you can achieve this, and it's quite trivial to do so! \n",
+    "\n",
+    "This tutorial *won't* cover some of the basics when it comes to nbdev, and instead comes with the understanding you know how to navigate nbdev (such as what are directives, `export`, etc). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba8290bf-a950-45ae-8eec-b2b0f903b57a",
+   "metadata": {},
+   "source": [
+    "## Getting started, how does `nbdev` make this easy?\n",
+    "\n",
+    "First, let's write out some problem we want to achieve here. For the sake of simplicity, let's say instead of doing:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc8b0d1f-91b4-4dc9-9cce-860dbc4e003f",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "::: {.column-margin}\n",
+    "Some text\n",
+    ":::\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da68881d-c1a7-421b-83b2-1b694bafb6fe",
+   "metadata": {},
+   "source": [
+    "We wanted a shorter way to write this out, making use of how nbdev and quarto writes their *directives*? \n",
+    "\n",
+    "This tutorial will show how to end up with an API that looks like:\n",
+    "\n",
+    "```\n",
+    "#| .column-margin\n",
+    "\n",
+    "Some text\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "391cc508-468b-432c-b1e6-c631fdadce87",
+   "metadata": {},
+   "source": [
+    "And this will include cases where a div should be put across *multiple* cells as well, by specifying a `start` and an `end."
```

Comment:
There is one missing end tick (`)
```suggestion
    "And this will include cases where a div should be put across *multiple* cells as well, by specifying a `start` and an `end`."
```